.gitignore

```diff
@@ -19,7 +19,6 @@ node_modules
 
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
```

Comment:
why did you delete this one ?

Reply:
ok, when using code spell checker, the settings of the current user languages are being set here. so to prevent conflicts with other translators, I skipped this file.